code/modules/reagents/chemistry/holder.dm

```diff
@@ -211,36 +211,44 @@ var/const/INGEST = 2
 			continue
 		if(!M)
 			M = R.holder.my_atom
+
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
 			//Check if this mob's species is set and can process this type of reagent
 			var/can_process = 0
```

Comment:
I forgot to mention this also handles OD and addictions.

If we remove the can_process stuff (with PROCESS_SYN and whatnot), you end up with IPCs addicted to drugs they can't process and humans OD'ing on Surge. Which, I hope goes without saying, is bad for a number of reasons.